packages/integration-sdk-core/src/types/jobState.ts

```diff
@@ -4,6 +4,10 @@ export interface GraphObjectFilter {
   _type: string;
 }
 
+export interface GraphObjectLookupKey extends GraphObjectFilter {
+  _key: string;
+}
+
```

Comment:
I think this should not extend `GraphObjectFilter` because it will only get a single entity, which is by definition the unique combination of `_type` and `_key` - we won't expect to support other properties when `GraphObjectFilter` is expanded to support more properties.

Reply:
Fair!